tcf_website/templates/about/terms.html

```diff
@@ -71,8 +72,8 @@ <h4>User Conduct</h4>
         <li>upload, post, transmit, share or otherwise make available any material that contains software viruses or any other computer code, files or programs designed to interrupt, destroy or limit the functionality of any computer software or hardware or telecommunications equipment;</li>
         <li>intimidate or harass another;</li>
         <li>upload, post, transmit, share, store or otherwise make </li>available content that would constitute, encourage or provide instructions for a criminal offense, violate the rights of any party, or that would otherwise create liability or violate any local, state, national or international law;</li>
-        <li>use or attempt to use another's account, service or system without authorization from us, or create a false identity on the Service or the Site.</li>
-        <li>upload, post, transmit, share, store or otherwise make available content that, in our sole judgment , is objectionable or which restricts or inhibits any other person from using or enjoying the Site, or which may expose us or our users to any harm or liability of any type.</li>
+        <li>use or attempt to use another's account, service or system without authorization from us, or create a false identity on the Service or the Site;</li>
+        <li>upload, post, transmit, share, store or otherwise make available content that, in our sole judgment , is objectionable or which restricts or inhibits any other person from using or enjoying the Site, or which may expose us or our users to any harm or liability of any type;</li>
```

Comment:
Not your change, but there's an empty space after `sole judgement` - might as well fix it while you fix Pylint?